ai/src/ai/common/example.py

```diff
@@ -88,6 +91,10 @@ def get(self, id: str) -> T:
       if os.path.exists(raw_output_path):
         with open(raw_output_path) as f:
           entity.output.raw_output = f.read()
+      udiff_output_path = os.path.join(dir_path, "udiff.txt")
```

Comment:
Fo raw_output.txt, I'm wondering if we can rename that at some point. Since I think this one contains the git patch diff. And I guess technically also could be the full output (but don't think we have any examples of that).

Or I guess the raw_output.txt can also can udiff output?

Seems like you're leaning toward going with unified diff format, so may be moot in terms of having the flexibility. I think the unified diff format does seem a bit better especially for multiple diffs.

Reply:
Yeah, I've been wondering about this too. I think there could be some value in having output that's not strictly code (i.e. diff/output) where it's some kind of preamble as a planning stage (e.g. consider the different interpretations of what the user wanted in cases where it's unclear). 

I do think overall udiff is better than our existing diff format because the models, particularly llama, seem to understand it better / prefer it. After some more experimentation, we could choose to just settle on udiff.